src/react-query/queryClient.ts

```diff
@@ -6,7 +6,7 @@ import { PersistQueryClientOptions } from '@tanstack/react-query-persist-client'
 export const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
-      cacheTime: Infinity,
+      cacheTime: 1000 * 60 * 60 * 24 * 7,
```

Comment:
It isn't necessary but would be cool to make it read without having to figure out how many days weeks hours it is
```
const SEVEN_DAYS = 1000 * 60 * 60 * 24 * 7;
```
and then
```suggestion
      cacheTime: SEVEN_DAYS,
```

Reply:
Love it, thank you